src/main/java/com/microfocus/application/automation/tools/model/SummaryDataLogModel.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
The new files should be created under lr folder because they are yet to be serialized in any build.